components/script/dom/window.rs

```diff
@@ -1356,7 +1402,9 @@ impl Window {
     }
 
     pub fn get_url(&self) -> Url {
-        (*self.Document().url()).clone()
+        let doc = self.Document();
+        let val = (*doc.url()).clone();
+        val
```

Comment:
I am sure to say that there is no workaround unless we add a method to `Document` that returns a clone of the `Document.url`.